doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,723 @@
+but IMPROVER can also generate forecasts as a set of percentiles, 
+for which the metadata will differ slightly. 
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+Dimensions
+^^^^^^^^^^
+
+The ``threshold`` dimension is replaced by ``percentile`` dimension
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 2-6
+    :emphasize-lines: 2
+ 
+percentile
+    The dimension for the coordinate describing the percentile values. 
+
+Main percentile variable 
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the example here, the main variable is a 12 hour maximum temperature,
+but representing the Nth percentile of the temperature forecast probability distribution
+at each point. 
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 8-14
+
+The variable attributes are the same as the probability example
+previously described in detail, except: 
+
+standard_name
+    Can be used rather than a ``long_name``, as ``air_temperature``
+    exists as a descriptive name in the `CF Standard Name`_ list. 
+
+cell_methods
+    Used to describe statistical processing applied to the quantity
+    no longer requires the additional non-standardized part, 
+    ``(comment: of air_temperature)``, 
+    as they now refer to the main variable that is a maximum temperature over 12 hours. 
+
+Percentile coordinate variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the main variable provides the air temperature at a set of percentiles,
+a further dimensional coordinate variable is required to allow the data to be 
+fully interpreted. This holds the set of percentile values. 
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 24-26, 72-74
+
+The variable attributes are:
+
+long_name
+   ``percentile`` which is used for all IMPROVER percentile variables. 
+   It is not in the `CF Standard Name`_ list.
+
+units
+    The units here are a percentage. 
+
+
+Spot data 
+*********
+
+Broadly, the spot data parallels the form of the gridded data, 
+both probabilities and percentiles are produced,
+but there are some notable differences because of the nature of the data.
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+Global attributes 
+^^^^^^^^^^^^^^^^^
+
+Only the title is different for the spot data. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 60-66
+    :emphasize-lines: 6
+
+Dimensions
+^^^^^^^^^^
+
+The dimensions for the horizontal coordinate variables have been replaced by
+a dimension for an index coordinate for the sites, 
+and a dimension for the WMO identifier string length has been added. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 2-7
+
+spot_index
+    The dimension for the index for the set of sites. 
+
+string5 / string8
+    Just constants used to dimension the character length of the string variable
```

Comment:
See previous comments on starting sentence with "Just..."

Reply:
And also changed.